### PR TITLE
Merge SAP portal branch (cap-portal)

### PR DESCRIPTION
**Pull request by polybuildr**

### CODE REVIEW

**Comment by polybuildr** [comment]
@erilyth could you please review the changes made from b069ddb - 4a67344 (inclusive) to just confirm that I didn't make any stupid mistakes?

**Comment by erilyth** [comment]
Everything in b069ddb - 4a67344 seems fine

File: sql/create_table_sap_ambassadors.sql [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+  `phone_number` varchar(32) NOT NULL,
+  `college` varchar(128) NOT NULL,
+  `program_of_study` varchar(128) NOT NULL,
+  `year_of_study` char(8) NOT NULL,
+  `facebook_profile_link` varchar(64) NOT NULL,
+  `why_apply` text NOT NULL,
+  `about_you` text NOT NULL,
+  `organised_event` text,
+  `registration_time` timestamp NOT NULL DEFAULT CURRENT_TIMESTAMP
+) ENGINE=InnoDB DEFAULT CHARSET=latin1;
```
Comment (nisargjhaveri) [inline sql/create_table_sap_ambassadors.sql]: We should make effort to always use utf8 charset.

**Comment by nisargjhaveri** [comment]
Scrutinizer still have some patches for code styling.

**Comment by polybuildr** [comment]
I looked at the patches for all files introduced in this PR.

Apart from the `if (! isset` (the space), all the other patches don't seem valid. And the `isset` one was intentional, so I didn't make that change either.

**Comment by nisargjhaveri** [comment]
https://scrutinizer-ci.com/g/felicity-iiith/felicity16-website/inspections/744fd752-7c02-49e0-9f9f-ad5e1a91e211/patches?status=new 

Apart from the first one everything seems valid.

**Comment by nisargjhaveri** [comment]
Add the final schema somewhere?

**Comment by polybuildr** [comment]
Yes, adding.

Vivek Ghaisas

**Comment by nisargjhaveri** [comment]
Looks fine. Merging.

**Comment by polybuildr** [comment]
Finally! :D

**Pull request merged**